rep2.md

```diff
@@ -55,7 +55,7 @@ Every REP (including branches) carries a different number. New versions of the s
 ### REP template
 Each REP MUST customize and include this header:
 ````
-```
+```Draft
```

Comment:
Is this intended? I know that you can put syntax highlighting here like `diff` or `bash`, but never saw draft